### PR TITLE
DS-279 Implement 1st batch of icons (WIP)

### PR DESCRIPTION
**Pull request by david-cho-lerat-sonarsource**

### CODE REVIEW

File: src/components/icons/IconWrapper.tsx [hunk 1]
```diff
@@ -34,34 +34,17 @@ function IconBase(props: Readonly<IconProps & { children: React.ReactNode }>) {
...
+  font-size: calc(1em * (18 / 14));
   font-style: normal;
-  line-height: 1;
-  letter-spacing: normal;
-  text-transform: none;
-  white-space: nowrap;
-  word-wrap: normal;
-  direction: ltr;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
```
Comment (gregaubert) [inline src/components/icons/IconWrapper.tsx:47]: Should we keep these for improved rendering on compatible browsers?

File: src/components/icons/IconWrapper.tsx [hunk 2]
```diff
@@ -34,34 +34,17 @@ function IconBase(props: Readonly<IconProps & { children: React.ReactNode }>) {
...
-  text-rendering: optimizeLegibility;
+  font-weight: normal;
+  height: calc(1em * (20 / 18));
+  line-height: calc(1em * (20 / 18));
+  text-align: center;
+  width: calc(1em * (20 / 18));
```
Comment (gregaubert) [inline src/components/icons/IconWrapper.tsx:43]: this works with display: inline ??

File: stories/Icons-stories.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+  & span {
+    margin-top: 8px;
+  }
+`;
+
+const baseStyle = {
+  display: 'flex',
+  columnGap: '4px',
+};
```
Comment (jeremy-davis-sonarsource) [inline stories/Icons-stories.tsx:86]: I'd like it to work without a flex container.
It seems that if you make it `vertical-align: bottom`, it aligns well

File: src/components/icons/IconWrapper.tsx (hunk 2 above)
Comment (jeremy-davis-sonarsource) [reply src/components/icons/IconWrapper.tsx:43]: no, `inline-block` is needed. And as I mentioned, making it `vertical-align: bottom` aligns it without the need for a flex container

**Review by jeremy-davis-sonarsource — DISMISSED** [review]

File: src/components/icons/IconWrapper.tsx (hunk 2 above)
Comment (jeremy-davis-sonarsource) [reply src/components/icons/IconWrapper.tsx:46]: Whoops, still missing `display: inline-block`

**Review by gregaubert — APPROVED** [review]
Nice! :)

**Pull request merged**